hat/assets/js/apps/Iaso/domains/workflows/details.tsx

```diff
@@ -143,6 +143,23 @@ export const Details: FunctionComponent<Props> = ({ router }) => {
         setIsFollowUpOrderChange(true);
     }, []);
 
+    const handleResetFollowUpsOrder = useCallback(() => {
+        if (workflowVersion?.follow_ups) {
+            const newFollowUps = orderBy(
+                workflowVersion.follow_ups,
+                [f => f.order],
+                ['asc'],
+            );
+            setFollowUps(
+                newFollowUps.map(followUp => ({
+                    ...followUp,
+                    accessor: followUp.id,
+                })),
+            );
```

Comment:
This logic is copy pasted from an existing useEffect. You should put it in a separate function